producers-commissioning/src/main/java/uk/gov/moj/sdt/producers/comx/config/bulkfeedback/BulkFeedbackConfigCOne.java

```diff
@@ -29,7 +30,7 @@ private BulkSubmission bulkSumissionC00000001() {
     }
 
     @Bean("createIndividualRequestC0000001")
-    public void invokingBean() {
+    public void invokingBean() throws ClassNotFoundException, NoSuchMethodException, InvocationTargetException, IllegalAccessException {
         MethodInvokingFactoryBean methodInvokingFactoryBean = new MethodInvokingFactoryBean();
         methodInvokingFactoryBean.setTargetObject(bulkFeedbackFactory());
         methodInvokingFactoryBean.setTargetMethod("createIndividualRequest");
```

Comment:
Method name should be "createIndividualRequests" (note the "s" at the end).

To minimise the chance of this happening again it might be an idea to create a parent class for the BulkFeedbackConfig classes that has public constants defined for the two methods.

Reply:
done